### PR TITLE
Rewrite myuplink docs for new quality scale

### PR DESCRIPTION
**Pull request by astrandb**
## Proposed change

Update documentation to take a step towards the ultimate goal - Platinum grade

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Significantly revised myUplink integration documentation for enhanced clarity on functionality, including equipment control capabilities.
	- Elaborated on account connection and data retrieval processes, noting variability based on equipment type.
	- Added a reminder about the necessity of a valid myUplink subscription for equipment control.
	- Introduced new sections for data updates, troubleshooting common login issues, and known limitations regarding API data mapping.
	- Revised terminology in prerequisites for better understanding, including updates to "Application Name" and "Callback URL."

### CODE REVIEW

File: source/_integrations/myuplink.markdown [hunk 1]
```diff
@@ -55,3 +59,22 @@ Internal examples: `http://192.168.0.2:8123/auth/external/callback`, `http://hom
...
+
+## Data updates
+
+The integration will poll the API for data every 5 minutes. This polling interval is designed to work within the rate limits of myUplink APIs while providing timely updates.
+
+## Troubleshooting
+
+### Can't log in to myUplink API
+
+- Make sure that you have entered the application credentials correctly. A common problem is that leading or trailing spaces are included in the entered credential strings. You may have to delete the application credentials from Home Assistant and install the integration again to get everything right.
```
Comment (klaasnicolaas) [inline source/_integrations/myuplink.markdown]:
```suggestion
{% details "Can't log in to myUplink API" %}
Make sure that you have entered the application credentials correctly. A common problem is that leading or trailing spaces are included in the entered credential strings. You may have to delete the application credentials from Home Assistant and install the integration again to get everything right.
{% enddetails %}
```
You could do this in a details block and move the troubleshooting and remove instructions to the bottom of the page.

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by klaasnicolaas** [comment]
Converting back to draft, nothing has been done with the feedback yet.

./klaas

**Comment by astrandb** [comment]
Sorry, forgot to push the changes.

File: source/_integrations/myuplink.markdown [hunk 2]
```diff
@@ -55,3 +59,22 @@ Internal examples: `http://192.168.0.2:8123/auth/external/callback`, `http://hom
...
+
+## Troubleshooting
+
+{% details "Can't log in to myUplink API" %}
+Make sure that you have entered the application credentials correctly. A common problem is that leading or trailing spaces are included in the entered credential strings. You may have to delete the application credentials from Home Assistant and install the integration again to get everything right.
+{% enddetails %}
+
+{% include integrations/remove_device_service.md %}
+
+After deleting the integration, go to the myUplink developer [site](https://dev.myuplink.com/apps) and remove the credentials unless you will use them again.
```
Comment (klaasnicolaas) [inline source/_integrations/myuplink.markdown]:
```suggestion
## Remove integration

After removing the integration, go to the myUplink [developer site](https://dev.myuplink.com/apps) and remove the credentials unless you will use them again.

{% include integrations/remove_device_service.md %}
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
Another small adjustment for the removal instructions

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_integrations/myuplink.markdown [hunk 3]
```diff
@@ -55,3 +59,24 @@ Internal examples: `http://192.168.0.2:8123/auth/external/callback`, `http://hom
...
+- The integration makes the best effort to map data-points from the API to relevant entities in Home Assistant. However, some sensors may not appear for certain heat-pump models, or in other cases, numerous irrelevant entities might be created. Please create an issue on GitHub and include a diagnostic download file from your installation if you believe that the mapping can be improved.
+- Entity names are available in English and cannot be automatically translated by Home Assistant. The reason is that the names are defined by the API and can be changed by updates of the API or the firmware in the appliance. However, most entity names are self-explanatory, e.g., "Room temperature (BT50)".
+
+## Troubleshooting
+
+{% details "Can't log in to myUplink API" %}
+Make sure that you have entered the application credentials correctly. A common problem is that leading or trailing spaces are included in the entered credential strings. You may have to delete the application credentials from Home Assistant and install the integration again to get everything right.
+{% enddetails %}
+
+## Remove integration
```
Comment (c0ffeeca7) [inline source/_integrations/myuplink.markdown]:
```suggestion
## Removing the integration
```
I know this is from the example. But I wanted to use the -ing form, like other titles that introduce procedural content.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @astrandb :+1:

**Review by klaasnicolaas — APPROVED** [review]
Thnx for the PR! 🥇

./Klaas

**Pull request merged**